exonum/src/blockchain/schema.rs

```diff
@@ -276,10 +301,7 @@ impl<T: Access> Schema<T> {
 
     /// Returns the `state_hash` table for core tables.
     pub fn state_hash(&self) -> Vec<Hash> {
-        vec![
-            self.consensus_config_entry().object_hash(),
-            self.transaction_results().object_hash(),
-        ]
+        vec![self.consensus_config_entry().object_hash()]
```

Comment:
Shouldn't "call_errors" be included here? (not sure how this interacts with #1553).

Reply:
`call_errors` are specific per block, thus are not aggregated in the `state_hash` now, neither will they after automatic state aggregation is implemented.